### ISSUE TITLE
Build error using platform 0.21

### ISSUE THREAD

**Issue by slithzerikai** [SAP/fundamental-ngx#3066]
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.

Relatively new to angular development, so started a project using ngx-rocket generator and added fundamental core and platform libraries (see dependencies below) and all was fine on 0.20.0 until I upgraded to 0.21.0.

With 0.21.0 build throws an error:

ERROR in node_modules/@fundamental-ngx/platform/lib/components/form/switch/switch/switch.component.d.ts:39:29 - error TS2307: Cannot find module '../../../../../../../../dist/libs/core' or its corresponding type declarations.

39     _contentDensity: import("../../../../../../../../dist/libs/core").ToolbarSize;
                               ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

I can edit that file locally by hand to remove that import, and the problem goes away. 

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)

Angular 10 
Fundamental (platform) 0.21.0

#### If this is a bug, please provide steps for reproducing it.
Error happens on build:

npm run env -s && ng build --prod

node version: v12.16.2
npm version: 6.14.8

The error also happens on 'start' (npm run env -s && ng serve).

#### Please provide relevant source code if applicable.

package.json dependencies:

  "dependencies": {
    "@angular/animations": "^10.0.11",
    "@angular/cdk": "^10.1.3",
    "@angular/common": "^10.0.11",
    "@angular/compiler": "^10.0.11",
    "@angular/core": "^10.0.11",
    "@angular/forms": "^10.0.11",
    "@angular/localize": "^10.0.11",
    "@angular/platform-browser": "^10.0.11",
    "@angular/platform-browser-dynamic": "^10.0.11",
    "@angular/router": "^10.0.11",
    "@angular/service-worker": "^10.0.11",
    "@fortawesome/fontawesome-free": "^5.14.0",
    "@fundamental-ngx/core": "^0.21.0",
    "@fundamental-ngx/platform": "^0.21.0",
    "@ng-bootstrap/ng-bootstrap": "^6.0.3",
    "@ngx-translate/core": "^12.0.0",
    "@sap-theming/theming-base-content": "^11.1.21",
    "bootstrap": "^4.1.1",
    "fundamental-styles": "^0.11.2",
    "lodash": "^4.17.20",
    "moment": "^2.24.0",
    "rxjs": "^6.5.5",
    "zone.js": "^0.10.3",
    "core-js": "^3.6.5",
    "popper.js": "^1.16.1",
    "tslib": "^1.11.2",
    "typescript": "^3.7.5"
}

#### Is there anything else we should know?

I've been messing around with dependencies trying to find out if the issue is related to the version of angular or something else to no avail. But I am far from knowledgeable here. I've been unable to reproduce the issue in stackblitz either.

Very likely an error on my side, but I cannot figure out where the problem is.

**Comment by salarenko** [comment]
Hey @slithzerikai this is a little bug in Platform. It has been fixed here #3040 and hopefully will be released today as a patch to v21.

**Comment by salarenko** [comment]
Sorry for the delay. Should be working fine now with [v0.21.2](https://www.npmjs.com/package/@fundamental-ngx/platform)
